src/routes/(auth)/reset-password/+page.svelte

```diff
@@ -63,8 +61,9 @@
 					<a
 						href="/login"
 						class="underline text-blue-600 hover:text-blue-800 visited:text-purple-600"
-						>Back to login page</a
 					>
```

Comment:
Can we quickly talk about how to do the code formatting? I like yours way better, somehow running npm run format always returns these multi line blocks for me. like this:

<a
						href="/login"
						class="underline text-blue-600 hover:text-blue-800 visited:text-purple-600"
						>Back to login page</a
					>

Reply:
for me too! I thought it was the prettier default config for Svelte. Or at least, it is according to ChatGPT. I don't like it either way

Reply:
yepp, I saw that
Prettier isn't the smartest, so if you leave the text next to the ending/opening tag it won't interpret it properly and won't format it right
the only thing I did is add a line before and after the text, and then ran `npm run format`, that fixed it :pizza: